it/__init__.py

```diff
@@ -265,3 +267,27 @@ def setup_module():
 def teardown_module():
     ES_METRICS_STORE.stop()
     remove_integration_test_config()
+
+
+# ensures that a fresh log file is available
+@pytest.fixture(scope="function")
+def fresh_log_file():
+    cfg = ConfigFile(config_name=None)
+    log_file = os.path.join(cfg.rally_home, "logs", "rally.log")
+
+    if os.path.exists(log_file):
+        bak = os.path.join(tempfile.mkdtemp(), "rally.log")
+        shutil.move(log_file, bak)
+        yield log_file
+        # append log lines to the original file and move it back to its original
+        with open(log_file) as src:
+            with open(bak, "a") as dst:
+                dst.write(src.read())
+        shutil.move(bak, log_file)
+    else:
+        yield log_file
```

Comment:
Not related to this pull request, but it would be nicer to use a temporary directory rather than moving everything in place.